openwisp_monitoring/monitoring/tests/test_admin.py

```diff
@@ -0,0 +1,59 @@
+from django.test import TestCase
+from django.urls import reverse
+from swapper import load_model
+
+from . import TestMonitoringMixin
+
+Notification = load_model('openwisp_notifications', 'Notification')
+
+
+class TestAdmin(TestMonitoringMixin, TestCase):
+    def test_action_disable_notifications(self):
+        m = self._create_general_metric(name='test')
+        self._create_threshold(metric=m, operator='>', value=90, seconds=0)
+        path = reverse('admin:monitoring_metric_changelist')
+        self.client.force_login(self._get_admin())
+        post_data = {
+            '_selected_action': [m.pk],
+            'action': 'disable_notifications',
+            'csrfmiddlewaretoken': 'test',
+            'confirmation': 'Confirm',
+        }
+        response = self.client.post(path, post_data, follow=True)
+        m.refresh_from_db()
+        self.assertFalse(m.notifications)
+        self.assertEqual(response.status_code, 200)
+        m.write(99)
+        self.assertFalse(m.is_healthy)
+        self.assertEqual(Notification.objects.count(), 0)
+
+    def test_action_enable_notifications(self):
+        m = self._create_general_metric(name='test', notifications=False)
+        self._create_threshold(metric=m, operator='>', value=90, seconds=0)
+        path = reverse('admin:monitoring_metric_changelist')
+        self.client.force_login(self._get_admin())
+        post_data = {
+            '_selected_action': [m.pk],
+            'action': 'enable_notifications',
+            'csrfmiddlewaretoken': 'test',
+            'confirmation': 'Confirm',
+        }
+        response = self.client.post(path, post_data, follow=True)
+        m.refresh_from_db()
+        self.assertTrue(m.notifications)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Instead of using `self.client.post(...follow=True)` and then asserting the status_code. Can you use [self.assertRedirects](https://docs.djangoproject.com/en/3.0/topics/testing/tools/#django.test.SimpleTestCase.assertRedirects)?